build.gradle

```diff
@@ -33,6 +33,8 @@ dependencies {
     compile("org.postgresql:postgresql:42.2.5")
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-thymeleaf'
     compile 'org.springframework.boot:spring-boot-starter-security'
+    // https://mvnrepository.com/artifact/org.hibernate/hibernate-search-orm
+    compile group: 'org.hibernate', name: 'hibernate-search-orm', version: '5.11.1.Final'
```

Comment:
Nooo, na to ještě mrknu. Protože postgres umí fulltext nativně.

Reply:
ok